packages/lesswrong/components/spotlights/SpotlightItem.tsx

```diff
@@ -115,37 +134,59 @@ const styles = (theme: ThemeType): JssStyles => ({
     },
     ...(isFriendlyUI ? {
       fontSize: 13,
+      fontWeight: 500,
       fontFamily: theme.palette.fonts.sansSerifStack,
-      color: theme.palette.grey[700],
+      color: theme.palette.text.alwaysWhite,
       marginTop: 8,
+      maxWidth: TEXT_WIDTH,
+      "& a": {
+        color: theme.palette.text.alwaysWhite,
+        textDecoration: "underline",
+        "&:hover": {
+          opacity: 0.8,
+          color: `${theme.palette.text.alwaysWhite} !important`,
+        },
+        "&:visited": {
+          color: theme.palette.text.alwaysWhite,
+        },
+      },
     } : {}),
   },
   title: {
     ...theme.typography.headerStyle,
-    fontSize: 20,
-    ...(isFriendlyUI ?
-      {fontWeight: 600} :
-      {fontVariant: "small-caps"}
+    ...(isFriendlyUI
+      ? {
+        fontSize: 22,
+        fontWeight: 700,
+        color: theme.palette.text.alwaysWhite,
+      }
+      : {
+        fontSize: 20,
+        fontVariant: "small-caps",
+        lineHeight: "1.2em",
+      }
     ),
-    lineHeight: "1.2em",
     display: "flex",
     alignItems: "center"
   },
   subtitle: {
     ...theme.typography.postStyle,
-    color: theme.palette.grey[700],
     ...theme.typography.italic,
     ...(isFriendlyUI ? {
       fontSize: 13,
+      fontWeight: 500,
       fontFamily: theme.palette.fonts.sansSerifStack,
+      color: theme.palette.text.alwaysWhite,
       marginTop: 8,
+      maxWidth: TEXT_WIDTH,
     } : {
+      color: theme.palette.grey[700],
       fontSize: 15,
       marginTop: -1,
     }),
   },
   startOrContinue: {
-    marginTop: isFriendlyUI ? 16 : 4,
+    marginTop: isFriendlyUI ? 0 : 4,
```

Comment:
Might need some margin here for the mobile version:

<img width="361" alt="Screen Shot 2024-04-24 at 2 16 12 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/016cbec1-4b3e-40b5-9815-9d48b52b0805">